src/components/Product/FilterData.js

```diff
@@ -0,0 +1,154 @@
+              <span>1000</span>
+            </div>
+            <input
+              type='range'
+              className='slider'
+              min='100'
+              max='1000'
+              onChange={(e) =>
+                dispatch({
+                  type: "PRICE_RANGE",
+                  payload: e.target.value,
+                })
+              }
+            />
+          </div>
+          <hr className='filterhr' />
+          {/* <div className='filters'>
+            <h1>Category</h1>
+            {state.categories.map((items) => {
+              return (
+                <>
+                  <ul>
+                    <li key={items.id}>
+                      <input
+                        type='checkbox'
+                        // checked={items.categoryName}
+                        onChange={() =>
+                          dispatch({
+                            type: "FILTER_BY_CATEGORY",
+                            payload: items.categoryName,
+                          })
+                        }
+                      />
+                      {items.categoryName}
+                    </li>
+                  </ul>
+                </>
+              );
+            })}
+          </div> */}
+
+          <div className='filters'>
+            <h1>Type</h1>
+            <ul>
+              <li>
+                <input
+                  type='checkbox'
+                  checked={state.filters.includeOutOfStock}
+                  onChange={() =>
+                    dispatch({
+                      type: "INCLUDE_OUT_OF_STOCK",
+                      payload: state.filters.includeOutOfStock,
+                    })
+                  }
+                />
+                Include Out of Stock
+              </li>
+              <li>
+                <input
+                  type='checkbox'
+                  checked={state.filters.fastDelivery}
+                  onChange={() =>
+                    dispatch({
+                      type: "FAST_DELIVERY",
+                      payload: state.filters.fastDelivery,
+                    })
+                  }
+                />{" "}
+                Fast Delivery
+              </li>
+            </ul>
+          </div>
+          <hr className='filterhr' />
+          {/* <div className='filters'>
+            <h1>Rating</h1>
+            <ul>
+              <li>
+                <input type='radio' name='rating' /> 5 start
+              </li>
+              <li>
+                <input type='radio' name='rating' /> 4 start
+              </li>
+              <li>
+                <input type='radio' name='rating' /> 3 start
+              </li>
+              <li>
+                <input type='radio' name='rating' /> 2 start
+              </li>
+              <li>
+                <input type='radio' name='rating' /> 1 start
+              </li>
+            </ul>
+          </div> 
+          <hr className='filterhr' />*/}
+
```

Comment:
Here as well.

Reply:
(some code is commented purposely) i have mentioned this in description as well